JumboCode/emerald-necklace-conservancy
# added map page to app

What Changes were made? 
Neya and I added rough implementation of map page / has links to the 7 different parks and a header called map

Type of change:
Added new feature (non-breaking change which adds functionality)

How to test changes:
If you go to page.tsx file within app folder and import MapPage from ./map/page and then use the <MapPage/> tag you should be able to test our page.

Screenshots:
<img width="814" alt="image" src="https://github.com/JumboCode/emerald-necklace-conservancy/assets/59243871/f80cd8ce-af71-4fa8-b20f-18b78f4f1e88">

<img width="809" alt="image" src="https://github.com/JumboCode/emerald-necklace-conservancy/assets/59243871/d714bd1f-7e0c-449a-bca2-d3273001e799">

<img width="903" alt="image" src="https://github.com/JumboCode/emerald-necklace-conservancy/assets/59243871/1e4877f9-b6fe-4241-aeb1-0f41252078af">

Checklist:
- [X] My code follows the spec of my ticket and completes what the ticket wants in its entirety
- [x] My code follows the style guidelines of the project and I have checked that lint passes
